hphp/runtime/ext/ext.h

```diff
@@ -26,21 +26,49 @@
 #include "hphp/facebook/extensions/phpmc/ext_phpmc.h"
 #endif
 
+#ifdef ENABLE_EXTENSION_ARRAY
 #include "hphp/runtime/ext/array/ext_array_idl.h"
```

Comment:
Actually, it looks like a lot of these includes need to be deleted from ext.h

Only extensions which are implemented with IDL need to be included from here.
I'll put up a diff to strip this file down as well.